tests/phpunit/data/blocks/fixtures/core__gallery__deprecated-4.server.html

```diff
@@ -0,0 +1,20 @@
+
+<figure class="wp-block-gallery columns-3 is-cropped alignwide wp-block-gallery-1">
+	<ul class="blocks-gallery-grid">
+		<li class="blocks-gallery-item">
+			<figure>
+				<img src="https://sergioestevaofolio.files.wordpress.com/2016/09/cropped-img_9054-1.jpg?w=190" alt="" data-id="1421" class="wp-image-1421"/>
+			</figure>
+		</li>
+		<li class="blocks-gallery-item">
+			<figure>
+				<img src="https://sergioestevaofolio.files.wordpress.com/2017/09/cropped-l1001498-1.jpg?w=580" alt="" data-id="1440" class="wp-image-1440"/>
+			</figure>
+		</li>
+		<li class="blocks-gallery-item">
+			<figure>
+				<img src="https://sergioestevaofolio.files.wordpress.com/2017/05/cropped-l1005945-2-2.jpg?w=580" alt="" data-id="1362" class="wp-image-1362 is-layout-flex"/>
```

Comment:
`is-layout-flex` doesn't look appropriate here as it's not on the other images. I am unsure if it has an effect.